cmd/rook/ceph/osd.go

```diff
@@ -286,17 +284,17 @@ func commonOSDInit(cmd *cobra.Command) {
 	clusterInfo.Monitors = mon.ParseMonEndpoints(cfg.monEndpoints)
 }
 
-func getLocation(clientset kubernetes.Interface, location string, topologyAware bool) (string, error) {
+func getLocation(clientset kubernetes.Interface, location string, pvcBacked bool) (string, error) {
 	locArgs, err := client.FormatLocation(cfg.location, cfg.nodeName)
 	if err != nil {
 		return "", fmt.Errorf("invalid location. %+v", err)
 	}
 	// use zone/region/hostname labels in the crushmap
-	if cfg.topologyAware {
+	if cfg.pvcBacked {
 		nodeName := os.Getenv(k8sutil.NodeNameEnvVar)
 		node, err := getNode(clientset, nodeName)
 		if err != nil {
-			return "", fmt.Errorf("topologyAware is set, but could not get the node: %+v", err)
+			return "", fmt.Errorf("could not get the node where the OSD is running on. %+v", err)
```

Comment:
Grammar: `could not get the node where the OSD is running` or `could not get the node on which the OSD is running`